test/e2e/operator/nmstate_install_test.go

```diff
@@ -130,6 +131,41 @@ var _ = Describe("NMState operator", func() {
 			})
 		})
 	})
+	Context("when cluser-reader exists", func() {
+		const (
+			clusterReaderRoleName = "cluster-reader"
+			testUserNamespace     = "default"
+			serviceAccountName    = "test-user"
+			testUserBindingName   = "test-user-binding"
+		)
+
+		BeforeEach(func() {
+			Expect(createClusterReaderCR(clusterReaderRoleName)).To(Succeed(),
```

Comment:
This probably fails on testing on Openshift as the cluster role exists already. But you could check in case of an error is it already existed (`errors.IsAlreadyExists(err)`)